extensions/Vendor/FRAUNHOFER_materials_pbr/README.md

```diff
@@ -0,0 +1,146 @@
+# FRAUNHOFER_materials_pbr
+
+## Contributors
+
+* Timo Sturm, Fraunhofer IGD, [@\_tsturm\_](https://twitter.com/\_tsturm\_)
+* Miguel Sousa, Fraunhofer IGD, [@mfportela](https://twitter.com/mfportela)
+* Maik Th&ouml;ner, Fraunhofer IGD, [@mthoener](https://twitter.com/mthoener)
+* Max Limper, Fraunhofer IGD, [@mlimper_cg](https://twitter.com/mlimper_cg)
+
+## Status
+
+Draft
+
+## Dependencies
+
+Written against the glTF 1.0 spec.
+
+## Overview
+
+The glTF 1.0 allows the definition of materials by instancing `techniques`. A `technique`, as defined in glTF 1.0, is a verbose description of shader parameters combined with actual shader code. Typically, shader code is engine specific and, as such, may not be used across systems which do not share the same rendering pipeline.
```

Comment:
To help better position `FRAUNHOFER_materials_pbr` in the glTF ecosystem, consider adding a sentence here about the [KHR_materials_common](https://github.com/KhronosGroup/glTF/tree/master/extensions/Khronos/KHR_materials_common) extension, which defines non-PBR materials.